daemon/daemon.go

```diff
@@ -258,14 +262,22 @@ func New() *Daemon {
 	if err != nil {
 		panic(err.Error())
 	}
-	repo := caps.NewRepository()
-	err = caps.LoadBuiltInTypes(repo)
+	capRepo := caps.NewRepository()
```

Comment:
Just as soon as the new CLI bits land and I can remove all of capabilities in one big swoop.